sql/core/src/main/scala/org/apache/spark/sql/jdbc/AggregatedDialect.scala

```diff
@@ -42,6 +42,18 @@ private class AggregatedDialect(dialects: List[JdbcDialect]) extends JdbcDialect
     dialects.flatMap(_.getJDBCType(dt)).headOption
   }
 
+  override def quoteIdentifier(colName: String): String = {
+    dialects.map(_.quoteIdentifier(colName)).head
```

Comment:
It's not necessary to call this on all the dialects, because only one is used. `dialects.head.quoteIdentifier(colName)`

Reply:
Thanks @srowen for the comment. I will change.